.github/workflows/tests.yml

```
@@ -2,10 +2,8 @@ name: Tests
 
 on:
   push:
+  workflow_dispatch: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)